app/hotels/src/singleHotel/hotelInformation/description/Facilities.js

```diff
@@ -85,12 +85,10 @@ export class Facilities extends React.Component<Props, State> {
 export default (createFragmentContainer(
   Facilities,
   graphql`
-    fragment Facilities_facilities on HotelFacilityConnection {
-      edges {
-        node {
-          id
-          name
-        }
+    fragment Facilities_facilities on HotelInterface {
```

Comment:
Should we rename the component to be `Amenities`?

Reply:
Yes, I am doing that as well 😊